osdk/deps/frame-allocator/src/cache.rs

```diff
@@ -15,22 +15,26 @@ cpu_local! {
 }
 
 struct CacheOfSizes {
-    cache1: CacheArray<1, 12>,
-    cache2: CacheArray<2, 6>,
-    cache3: CacheArray<3, 6>,
-    cache4: CacheArray<4, 6>,
+    cache1: CacheArray<1, 12, 1>,
+    cache2: CacheArray<2, 6, 1>,
+    cache3: CacheArray<3, 6, 1>,
+    cache4: CacheArray<4, 6, 1>,
 }
 
 /// A fixed-size local cache for frame allocation.
 ///
 /// Each cache array contains at most `COUNT` segments. Each segment contains
 /// `NR_CONT_FRAMES` contiguous frames.
-struct CacheArray<const NR_CONT_FRAMES: usize, const COUNT: usize> {
+/// The size of each frame is determined by `LEVEL`, with level 1 corresponding to the base page
+/// size, and higher levels being a function of `LEVEL`. See `mm::page_size`.
```

Comment:
Ideally, `mm::page_size` would be a link 
```
[`mm::page_size`]
```
Being able to click through is quite useful.